src/pikevm.rs

```diff
@@ -134,22 +134,118 @@ fn try_match_state<Input: InputIndexer, Dir: Direction>(
             _ => StateMatch::Fail,
         },
 
-        Insn::ByteSeq1(v) => nextinsn_or_fail!(cursor::try_match_lit(input, dir, &mut s.pos, v)),
```

Comment:
I think this deserves a `#[rustfmt::skip]`, the one-instruction-per-line is better IMO.